File: modelo/usuario.php
<?php
    class usuario {
        private $id;
        private $name;
        private $emailInstitucional;
        private $password;
        private $rol;

        public function getId(){
            return $this->id;
        }

        public function setId($id){
            $this->id = $id;
        }

        public function getName(){
            return $this->name;
        }

        public function setName($name){
            $this->name = $name;
        }

        public function getEmailInstitucional(){
            return $this->emailInstitucional;
        }

        public function setEmailInstitucional($emailInstitucional){
            $this->emailInstitucional = $emailInstitucional;
        }

        public function getPassword(){
            return $this->password;
        }

        public function setPassword($password){
            $this->password = $password;
        }

        public function getRol(){
            return $this->rol;
        }

        public function setRol($rol){
            $this->rol = $rol;
        }

        public function __toString(){
            return "Fabricante - Id: {$this->id} - Nome: {$this->name} - Email: {$this->emailInstitucional}";
        }
    }
?>

File: src/usuario/Usuario.java
package usuario;

public class Usuario {
    private int id;
    private String name;
    private String emailInstitucional;
    private String password;
    private String rol;

    public int getId() {
        return id;
    }
    public void setId(int id) {
        this.id = id;
    }
    public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name = name;
    }
    public String getEmailInstitucional() {
        return emailInstitucional;
    }
    public void setEmailInstitucional(String emailInstitucional) {
        this.emailInstitucional = emailInstitucional;
    }
    public String getPassword() {
        return password;
    }
    public void setPassword(String password) {
        this.password = password;
    }
    public String getRol() {
        return rol;
    }
    public void setRol(String rol) {
        this.rol = rol;
    }

    public String insert() {
        return "INSERT INTO usuario (nome, email, senha, rol) VALUES ('" + name + "', '" + emailInstitucional + "', '" + password + "', '" + rol + "'); \n";
    }
    
}